package.json

```diff
@@ -24,15 +24,23 @@
   },
   "homepage": "https://github.com/graphcool/graphql-yoga",
   "scripts": {
-    "prepublish": "npm run build",
+    "prepublish": "yarn build",
     "build": "rm -rf dist && tsc -d",
     "lint": "tslint --project tsconfig.json {src,test}/**/*.ts && prettier-check --ignore-path .gitignore {src,.}/{*.ts,*.js}",
     "format": "prettier --write --ignore-path .gitignore {src,.}/{*.ts,*.js}",
-    "test": "npm run lint && npm run build"
+    "test": "yarn lint && yarn build && ava",
```

Comment:
I think that you can try `run-s lint build ava` here (not sure for `ava`, you have maybe to export this one)

Reply:
true. make a new PR ;)